test/verificationCode.spec.js

```diff
@@ -142,11 +142,6 @@ describe("sendVerificationCode()", () => {
 
     const payload = {
       channel: "sms",
-      phoneNumber: "+15558769098",
-      email: "user@example.com",
-      username: "new-by-sms",
-      name: "New User",
-      data: { attr: "custom-data" },
```

Comment:
Removed everything except for `channel` and asserted that it does not throw an error